meta-xfstk/recipes-xfstk/xfstk/xfstk_1.8.1.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Build XFSTK Downloader for recovery"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit cmake
+
+DEPENDS_${PN} = "g++ qtbase5 qtcreator build-essential devscripts libxml2-dev alien doxygen graphviz libusb-dev libboost-all-dev"
+
+SRC_URI = "https://github.com/Exoyds/xfstk/archive/1.8.2.zip"
+#SRC_URI += "file://xfstk.sh"
+
+S = "${WORKDIR}/xfstk"
+
+TARGET_CC_ARCH += "${LDFLAGS}"
+
+do_compile() {
+	export DISTRIBUTION_NAME=ubuntu12.04
```

Comment:
Yeah, iirc the code requires libs that are not in ubuntu 20.04. So we could use patches to the source that fix that?

Reply:
The AurelienBailers XFSTK should work on 20.04 and I have built it without any issues, just needed to install one additional package - qtbase5.

Also looking at https://github.com/saaguirre/xfstk-code/commit/1bcd3bcf396aa96cb25115b62ac66ac9b9eb5373 it seems that the ubuntu12.04 flag just specifies two versions of libraries. I could have a look at updating that to avoid any confusion with the build flags, or maybe omit the ubuntu version all together and just have a ubuntu flag?